src/layouts/CenterLayout.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box, styled } from '@mui/material';
+import { Outlet } from 'react-router-dom';
+
+const BoxStyled = styled(Box)`
+    position: fixed;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+
+    padding: ${({ theme }) => theme.spacing(4)};
```

Comment:
These paddings are not equal to the ones applied for the Card. This Box is a container for the card, so paddings here limit the space for the card